src/Microsoft.Health.Dicom.Api/Features/Routing/UrlResolver.cs

```diff
@@ -129,6 +129,24 @@ public Uri ResolveRetrieveInstanceUri(InstanceIdentifier instanceIdentifier)
         return RouteUri(routeName, routeValues);
     }
 
+    /// <inheritdoc />
+    public Uri ResolveUpdateWorkitemUri(string workitemInstanceUid)
```

Comment:
Do we need this or should we return the resource url, similar to retrieve workitem url

Reply:
Updated.